src/obb/server/interactors/create_lobby_game.clj

```diff
@@ -0,0 +1,19 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.server.interactors.create-lobby-game
```

Comment:
Regarding the game creation, I just didn't see all the different scenarios, and the ones I did, I just pictured as small variations of this one, that would be solved by the same interactor. Nevermind.

Yeah, I must confess that I struggle about using the same language. An interactor was a concept created to name an object that represents the the top-level use cases. The name even suggests that it is really an entity, a subject and not an `interaction`. When I try to think what I would call to something like this, the best names I come up with for the namespaces are really `interactions` and `use-cases`. I don't particularly love any of them and that why I started thinking that maybe it should just be the `core` namespace, in the sense that it contains the core functionality of the game. But I guess that someone looking at a Compojure application would expect to see the `core` namespace booting up the server just like you said, so disregard that.

As a side note, I also wanted to check this PR before advancing more in mine because I also struggled with the naming of the data access components. I didn't want to use gateways or repositories because they carry all the semantics of an object with them and, in my opinion, using different vocabulary helps us tear apart the different implementations in our heads and that is a good thing.